src/app/public/modules/action-buttons/action-buttons.component.html

```diff
@@ -6,22 +6,38 @@
       (keyup)="onKeyUp($event)">
     </sky-search>
   </div>
-  <sky-row>
-    <nav class="stache-action-buttons-nav-container">
-      <sky-column screenSmall="6" screenMedium="4" screenLarge="3" *ngFor="let route of filteredRoutes">
-        <a stacheRouterLink="{{route.path}}" fragment="{{route.fragment}}" class="stache-action-button-override sky-action-button sky-btn-default sky-rounded-corners">
-          <div>
-            <sky-action-button-icon *ngIf="route.icon" iconType="{{route.icon}}">
-            </sky-action-button-icon>
-            <sky-action-button-header>
-              {{route.name}}
-            </sky-action-button-header>
-          </div>
-          <sky-action-button-details>
-            {{route.summary}}
-          </sky-action-button-details>
-        </a>
-      </sky-column>
-    </nav>
-  </sky-row>
+
+  <nav>
+    <sky-fluid-grid
+      [disableMargin]="true"
+    >
+      <sky-row>
+        <sky-column *ngFor="let route of filteredRoutes"
+          screenLarge="3"
+          screenMedium="4"
+          screenSmall="6"
+        >
+          <a
+            class="stache-action-button-override"
+            fragment="{{route.fragment}}"
+            stacheRouterLink="{{route.path}}"
+          >
+            <sky-action-button>
+              <sky-action-button-icon *ngIf="route.icon"
+                iconType="{{route.icon}}"
+              >
+              </sky-action-button-icon>
+              <sky-action-button-header>
+                {{route.name}}
+              </sky-action-button-header>
+              <sky-action-button-details>
+                {{route.summary}}
+              </sky-action-button-details>
+            </sky-action-button>
+          </a>
+        </sky-column>
+      </sky-row>
+    </sky-fluid-grid>
+  </nav>
```

Comment:
☝️ Not a part of the original scope of the PR, but I had already done the work to add the proper `sky-fluid-grid` template tags, so I just left it in here. There should otherwise be no functional changes to the template.

Reply:
i believe you. but i'd like someone from SCD more familiar w/ what this comp did to confirm. sometimes  they notice lil css issues that i never catch. there's no preview link for this right? so theyll have to j/ pull your branch and serve?

Reply:
@Blackbaud-ColbyWhite We use this component in the Stache docs if you want to test with that SPA.

Reply:
I tested with the skyux2-docs site, and everything looked good. I'm also okay with removing this change since its a bit of a scope creep. Just let me know how you'd like to proceed.